### ISSUE TITLE
Participant tab in event creation has "Guests" tooltip

### ISSUE THREAD

**Issue by 4jNsY6fCVqZv** [elementary/calendar#542]
Compared to the other tabs, I feel that renaming the label to "Participants" would be more consistent here.

**Comment by mcclurgm** [comment]
I agree, we should standardize on one of them instead of having 2 names for the same thing. I assume that we're set on using "Participants" now?

**Comment by 4jNsY6fCVqZv** [comment]
Rename it to "Participants" would make sense to me too, yeah. I would like to work on an improvement here and I hope that the elementary team can give feedback on this.

**Comment by 4jNsY6fCVqZv** [comment]
Rename it to "Participants" would make sense to me too, yeah. I would like to work on an improvement here and hope that the elementary team can give feedback on what is desired.

**Comment by 4jNsY6fCVqZv** [comment]
Rename it to "Participants" would make sense to me too, yeah. I would like to work on an improvement here and hope that the elementary team can give feedback on what is desired.

**Comment by fabiozaramella** [comment]
I agree that renaming everything to "Participants" makes more sense in the context of an event, but we should wait for an opinion from the @elementary/ux team.
